### DISCUSSION TITLE
How do I get rid of a cloned repo, within another repo?

### DISCUSSION THREAD

**Discussion by rfalanga** [community/community#21633] [New to GitHub]
<p>I just made a boneheaded mistake. I was in a local repo, where the remote is on GitHub. (However, in this case I’ve not committed the change locally nor have I pushed it to GitHub.)</p>
<p>I wasn’t paying attention to the fact that I was <strong>already</strong> in a local repo. I went to GitHub to clone a repo I forked about 15 minutes previously. I got the URL and cloned it within the repo I was already in. Then, I realized the mistake I’ve made.</p>
<p>I’ve tried entering a <code>git rm -r (folder for the repo within the repo)</code>, but that failed. (Error was “fatal: pathspec did not match any files”).</p>
<p>How do I either undo the repo within a repo, or delete it from the repo I put it in?</p>

**Comment by kingthorin** [comment]
<p>Move the inner repo directory to a “proper” location?</p>
<p>I think you’re over thinking things.</p>

**Comment by rfalanga** [comment]
<p>I hadn’t thought of that. However, I’ve already cloned the repo again to a different, proper location. So, all I need to do is get rid of that inner repo.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Just do a regular deletion for your platform (e.g. <code>rm -rf</code> on a Unix-ish one). <code>git rm</code> doesn’t work because that second repo isn’t part of the enclosing one, so there’s nothing to remove from it.</p>
